CovertInfiltration/Src/CovertInfiltration/Classes/X2StrategyElement_DefaultComplications.uc

```diff
@@ -0,0 +1,97 @@
+
+class X2StrategyElement_DefaultComplications extends X2StrategyElement config(Infiltration);
+
+var config array<name> LootcrateMissions;
+
+static function array<X2DataTemplate> CreateTemplates()
+{
+	local array<X2DataTemplate> Complications;
+	
+	Complications.AddItem(CreateRewardInterceptionTemplate());
+
+	return Complications;
+}
+
+static function X2DataTemplate CreateRewardInterceptionTemplate()
+{
+	local X2ComplicationTemplate Template;
+
+	`CREATE_X2TEMPLATE(class'X2ComplicationTemplate', Template, 'Complication_RewardInterception');
+
+	Template.OnChainComplete = SpawnRescueMission;
+	Template.OnChainBlocked = DoNothing;
```

Comment:
Just not assign anything here, instead of noop functions. Also, get rid of said function